sklearn/utils/_metadata_requests.py

```diff
@@ -390,7 +392,8 @@ def _route_params(self, params):
                 message=(
                     f"[{', '.join([key for key in unrequested])}] are passed but are"
                     " not explicitly set as requested or not for"
-                    f" {self.owner}.{self.method}"
+                    f" {self.owner}.{self.method}. To set them, use"
+                    f" `set_{self.method}_request({{metadata}}={{boolean}})` on it."
```

Comment:
same as above, you could create a snippet which they can almost copy/paste here.